dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/metadata/objectbundle/hooks/IdentifiableObjectBundleHook.java

```diff
@@ -92,7 +93,7 @@ private void handleAttributeValuesNoDuplicates( IdentifiableObject identifiableO
             AttributeValue attributeValue = iterator.next();
 
             // if value null or empty, just skip it
-            if ( StringUtils.isEmpty( attributeValue.getValue() ) ||
+            if ( StringUtils.isEmpty( attributeValue.getValue() ) &&
```

Comment:
Is this correct? 
- We will get to the second condition only if the first one is true.
- First one is true only if it is null or an empty string
- In the second condition, we go into the method and there we use `attributeValue.getValue()` but this one can be null. Therefore, resulting in NullPointerException. Also, it will occur within the lambda so it will be swallowed.

Can you please revise this logic carefully? I feel something fishy there.